scripts/locust/service.py

```diff
@@ -89,6 +89,8 @@ def download_manifest(self):
         def stop(self):
             self.interrupt()
 
+    # FIXME: '/repository/specimens' changed to '/repository/samples'
```

Comment:
FIXMEs must be resolved or deferred to a Github ticket and the ticket must be linked here.